docs/domains/DataAccess/services/DataAcquisitionService/openapi.yml

```
@@ -973,6 +973,186 @@ paths:
           description: Not Found
         "500":
           description: Internal Server Error
+  /location-org-config:
+    get:
+      tags:
+      - location-org-config
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)